turtlesim/src/turtle.cpp

```diff
@@ -143,12 +161,13 @@ bool Turtle::update(double dt, QPainter& path_painter, const QImage& path_image,
 
   teleport_requests_.clear();
 
-  if (ros::WallTime::now() - last_command_time_ > ros::WallDuration(1.0))
-  {
-    lin_vel_ = 0.0;
-    ang_vel_ = 0.0;
-  }
-
+  // TODO: It makes error
+  // if ((nh_->now() - last_command_time_) > rclcpp::Duration(1.0))
```

Comment:
I ran into this issue - the constructor for rclcpp::Duration() with one argument expects the time in nanoseconds. It has a second constructor that takes two arguments, the first being seconds, and the second being nanoseconds. I made the following change in my code to get it to work:

`if (nh_->now() - last_command_time_ > rclcpp::Duration(1.0, 0))`

Reply:
Thank you for your comments! I modified that.